hydrolib/core/basemodel.py

```diff
@@ -778,31 +852,77 @@ def _get_identifier(self, data: dict) -> Optional[str]:
             return filepath.name
         return None
 
-    @property
-    def _relative_mode(self) -> ResolveRelativeMode:
-        """Gets the ResolveRelativeMode of this FileModel.
 
-        Returns:
-            ResolveRelativeMode: The ResolveRelativeMode of this FileModel
-        """
-        return ResolveRelativeMode.ToParent
+class DiskOnlyFileModel(FileModel):
+    """DiskOnlyFileModel provides a stub implementation for file based
+    models which are not explicitly implemented within hydrolib.core.
+
+    It implements the FileModel with a void parser and serializer, and a
+    save method which copies the file associated with the FileModel
+    to a new location if it exists.
+
+    We further explicitly assume that when the filepath is None, no
+    file will be written.
+
+    Actual file model implementations *should not* inherit from the
+    DiskOnlyFileModel and instead inherit directly from FileModel.
```

Comment:
I think you mean:
"[..] and instead inherit from SerializableFileModel"?

Reply:
Well not necessarily, if you want to create a different `FileModel` which might not read the data through a parser / serializer, you might want to inherit from `FileModel` instead of `SerializableFileModel`. It is probably unlikely, so I will update the docstring to reflect this.